system/handlers/formcontrols/OneToManyConfigurator.cfc

```diff
@@ -0,0 +1,41 @@
+component {
+
+	property name="presideObjectService" inject="presideObjectService";
+
+	public string function index( event, rc, prc, args={} ) {
+		var targetObject  = args.relatedTo ?: "";
+		var targetFk      = args.relationshipKey ?: args.sourceObject;
+		var useVersioning = Val( rc.version ?: "" ) && presideObjectService.objectIsVersioned( targetObject );
+		var hasSortOrder  = presideObjectService.getObjectProperties( targetObject ).keyExists( "sort_order" );
+		var orderBy       = hasSortOrder ? "sort_order" : "";
+		
+		if ( Len( Trim( args.savedData.id ?: "" ) ) ) {
+			var records       = presideObjectService.selectData(
+				  objectName       = targetObject
+				, filter           = { "#targetFk#"=args.savedData.id }
+				, selectFields     = [ "id", "label" ]
```

Comment:
what happens here when the target object uses a label field other than "label", or doesn't have a label field at all? We _can_ use:

```
[ "id", "${labelfield} as label" ]
```

But perhaps it makes sense just to select all fields by not supplying a select fields array at all?